lib/resque/worker.rb

```diff
@@ -858,5 +857,11 @@ def very_verbose=(value)
 
       @very_verbose = value
     end
+
+    private
+
+    def log_with_severity(severity, message)
```

Comment:
I wondered which approach you were gonna take. I didn't even think of this one. :+1: